crates/runtime/src/accelerated_table.rs

```diff
@@ -222,6 +227,19 @@ impl AcceleratedTable {
         Ok(())
     }
 
+    pub async fn update_refresh_sql(&self, refresh_sql: Option<String>) -> Result<()> {
```

Comment:
I'd make this a generic field updating function.

Reply:
Thank you - to be addressed as part of upcoming PR that will persist updated settings and will operate by Acceleration object that can be passed as parameter